modules/converting-machine-api-infra-machines.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="converting-machine-api-infra-machines_{context}"]
+= Converting infrastructure machines created using the Machine API
+
+[IMPORTANT]
+====
+Creating infra machines using the Machine API is now deprecated.
+====
+
+Infrastructure machines created using the Machine API had both the `worker` and `infra` labels applied. This duel label required you to taint the node so that compute workloads were not scheduled on it. Managing infra workloads using taints is no longer recommended. Machines created using the Machine API must be converted to use only the `infra` label and have a custom machine config pool manage it.
+
+.Procedure
+
+. List the infra nodes available in the cluster:
```

Comment:
Suggest, clarifying that this is to find nodes which are miss configured, I know its in the title of the doc, but having it here too will help.